src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,85 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  setCurrentPage: (value: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  setCurrentPage,
+}) => {
+  const countPagintion: number = Math.ceil(total / perPage);
+
+  const handlePageClick = (page: number) => {
+    setCurrentPage(page);
+  };
+
+  const plassPageClick = () => {
+    if (currentPage < countPagintion) {
+      setCurrentPage(currentPage + 1);
+    }
+  };
+
+  const minusPageClick = () => {
```

Comment:
Naming is strange 😄 `handlePrevPageClick` and `handleNextPageClick` suits better